http-request-response.sublime-syntax

```diff
@@ -112,12 +112,12 @@ contexts:
 
   content-type:
     - meta_content_scope: meta.headers.http-request-response
-    - match: (application/json|text/json){{content_type_sep}}
+    - match: (application/json|text/json|[\w.-]+/[\w.-]+\+json){{content_type_sep}}
```

Comment:
do you think it makes sense to introduce a variable for the `[\w.-]+/[\w.-]+\+` pattern, as we are already using it twice and may use it more in future too? :)

Reply:
Sure, why not. Added in 2912b9dedd2f3785e0951b09eb8e04428ebdba91.

I followed the existing convention to wrap things in a non-capturing group in variable definitions. Out of interest, what's the reason for using it as much as it is currently? For example in `multipart_form_data_boundary` it seems unnecessary, ditto `http_version`'s outer one.

Reply:
The idea is to make the variable a single regex token in case one wants to put a quantifier on it somewhere, or switch regex mode where the variable is referenced. It's just a best practise tip - you're right that it's not strictly necessary due to our usages. https://github.com/sublimehq/Packages/issues/757#issuecomment-544136278